docs/source/tts/datasets.rst

```diff
@@ -1,10 +1,113 @@
+   :file: data/datasets.csv
+   :align: right
+   :header-rows: 1
+
+Corpus-Specific Data Preprocessing
+----------------------------------
+NeMo implements model-agnostic data preprocessing scripts that wrap up steps of **downloading raw datasets, extracting files, and/or normalizing raw texts, and generating data manifest files**. Most scripts are able to be reused for any datasets with only minor adaptations. Most TTS models work out-of-the-box with the LJSpeech dataset, so it would be straightforward to start adapting your custom script from `LJSpeech script <https://github.com/NVIDIA/NeMo/blob/main/scripts/dataset_processing/tts/ljspeech/get_data.py>`_. For some models that may require supplementary data for training and validating, such as speech/text alignment prior, pitch, speaker ID, emotion ID, energy, etc, you may need an extra step of **supplementary data extraction** by calling `script/dataset_processing/tts/extract_sup_data.py <https://github.com/NVIDIA/NeMo/blob/main/scripts/dataset_processing/tts/extract_sup_data.py>`_ . The following sub-sections demonstrate detailed instructions for running data preprocessing scripts.
+
+LJSpeech
+~~~~~~~~
+* Dataset URL: https://keithito.com/LJ-Speech-Dataset/
+* Dataset Processing Script: https://github.com/NVIDIA/NeMo/blob/main/scripts/dataset_processing/tts/ljspeech/get_data.py
+* Command Line Instruction:
+
+.. code-block:: shell-session
+
+    $ python scripts/dataset_processing/tts/ljspeech/get_data.py \
+        --data-root <your_local_dataset_root> \
+        --whitelist-path <your_local_whitelist_filepath> \
+        or default nemo_text_processing/text_normalization/en/data/whitelist/lj_speech.tsv
+
+    $ python scripts/dataset_processing/tts/extract_sup_data.py \
+        --config-path ljspeech/ds_conf \
+        --config-name ds_for_fastpitch_align.yaml \
+        manifest_filepath=<your_path_to_train_manifest> \
+        sup_data_path=<your_path_to_where_to_save_supplementary_data>
+
+
+LibriTTS
+~~~~~~~~
+* Dataset URL: https://www.openslr.org/60/
+* Dataset Processing Script: https://github.com/NVIDIA/NeMo/blob/main/scripts/dataset_processing/tts/libritts/get_data.py
+* Command Line Instruction:
+
+.. code-block:: console
+
+    $ python scripts/dataset_processing/tts/libritts/get_data.py \
+        --data-root <your_local_dataset_root> \
+        --data-sets dev_clean
+
+    $ python scripts/dataset_processing/tts/extract_sup_data.py \
+        --config-path ljspeech/ds_conf \
+        --config-name ds_for_fastpitch_align.yaml \
+        manifest_filepath=<your_path_to_train_manifest> \
+        sup_data_path=<your_path_to_where_to_save_supplementary_data>
+
+
+
+HiFiTTS
+~~~~~~~
+The texts of this dataset has been normalized already. So there is no extra need to preprocess the data again. But we still need a download script and split it into manifests.
+
+* Dataset URL: http://www.openslr.org/109/
+* Dataset Processing Script: TBD
+* Command Line Instruction: TBD
```

Comment:
do you still need TBDs?

Reply:
I think so. We will have a data script for downlowning.